src/panels/profile/ha-panel-profile.ts

```diff
@@ -250,6 +250,7 @@ class HaPanelProfile extends LitElement {
           max-width: 600px;
           margin: 0 auto;
           padding-bottom: env(safe-area-inset-bottom);
+          overflow: hidden;
```

Comment:
I see this issue and that this fixes it, I just have a hard time understanding why it happens, and why only in rtl...